bundles/CoreBundle/src/Command/ThumbnailsImageCommand.php

```diff
@@ -240,8 +240,8 @@ private function fetchThumbnailConfigs(InputInterface $input, string $thumbnailC
         return $thumbnailsToGenerate;
     }
 
-    protected function getItemName(int $count): string
+    protected function getItemName(?int $count): string
     {
-        return $count == 1 ? 'thumbnail' : 'thumbnails';
+        return $count === 1 ? 'thumbnail' : 'thumbnails';
```

Comment:
you probably also want:

```php
if (null === $count) {
    return 'thumbnail(s)';
}
```

And the same in a couple of other places

Reply:
It would only be a cosmetical thing. I am not sure if we should more code just because of this.  if the count is null it will always be thumbnails which seems correct to me.

Reply:
Note necessarily, if it is `null` it just means `fetchItems()` returned a non-array `iterable`. Behind there could be 0, 1 or X items.

But yes ultimately it's only cosmetics, the same way pluralizing the name is :)

Reply:
You are totally right, but I think semantically it would not make a difference unless you really want to know if you got a non-array iterable, which should not be possible since fetchItems need to return an array? :)

Reply:
>which should not be possible since fetchItems need to return an array? :)

That's been changed in v2 to support the case where you don't know all the items beforehand, hence the change in signature 😄